pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/BrainRename.java

```diff
@@ -551,6 +552,10 @@ private void collectQuestionTemplates(List<Template> templates, QuestionDef ques
                 var numQuestion = (NumericQuestionDef) question;
                 templates.add(numQuestion.getPlaceholderTemplate());
                 break;
+            case DECIMAL:
+                var decQuestion = (DecimalQuestionDef) question;
+                templates.add(decQuestion.getPlaceholderTemplate());
+                break;
```

Comment:
This is more than just a forgotten `$` sign. Please update the PR description.

Reply:
Done